### PR TITLE
[FSDE-30409] Pedro add e2 e tests for carousel autoplay config

### PR DESCRIPTION
**Pull request by pedrotaglialatela-lumen**

### CODE REVIEW

File: cypress/integration/chi-custom-elements/carousel.spec.js [hunk 1]
```diff
@@ -216,4 +217,88 @@ describe('Carousel', () => {
...
+    it('Should have attribute autoplay', () => {
+      cy.get('@autoplay').should('have.attr', 'autoplay');
+    });
+
+    it('Should have attribute interval', () => {
+      cy.get('@autoplay').should('have.attr', 'interval', 3);
+    });
+
+    it('Should autoplay with 3 seconds interval', () => {
+      cy.visit('tests/custom-elements/carousel.html');
```
Comment (josecarlosgonzalezv) [inline cypress/integration/chi-custom-elements/carousel.spec.js:257]: Do we need this `cy.visit`?

File: cypress/integration/chi-custom-elements/carousel.spec.js [hunk 2]
```diff
@@ -216,4 +217,88 @@ describe('Carousel', () => {
...
+
+      cy.get('@autoplay').within(() => {
+        cy.get(`${CAROUSEL_DIRECTION_CONTROL_CLASS}.${CAROUSEL_PREVIOUS_CLASS}`)
```
Comment (josecarlosgonzalezv) [inline cypress/integration/chi-custom-elements/carousel.spec.js:260]: Does it work like this? If yes, change it in the other test cases

```suggestion
        cy.get(`.${CAROUSEL_PREVIOUS_CLASS}`)
```

File: cypress/integration/chi-custom-elements/carousel.spec.js [hunk 3]
```diff
@@ -216,4 +217,88 @@ describe('Carousel', () => {
...
+    });
+
+    it('Should have attribute no-button-controllers', () => {
+      cy.get('@no-button-controllers').should(
+        'have.attr',
+        'no-button-controllers'
+      );
+    });
+
+    it('should not have direction buttons controller', () => {
```
Comment (josecarlosgonzalezv) [inline cypress/integration/chi-custom-elements/carousel.spec.js:233]:
```suggestion
    it('Should not have direction buttons controller', () => {
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: cypress/integration/chi-custom-elements/carousel.spec.js (hunk 1 above)
Comment (pedrotaglialatela-lumen) [reply cypress/integration/chi-custom-elements/carousel.spec.js:257]: @josecarlosgonzalezv unfortunately this is needed to reset the carousel to the first position. 
Another options: 
- Move this to be the first test
- Implement a way to set the carousel position (with a huge cost)

File: cypress/integration/chi-custom-elements/carousel.spec.js (hunk 3 above)
Comment (pedrotaglialatela-lumen) [reply cypress/integration/chi-custom-elements/carousel.spec.js:233]: Ok!

File: cypress/integration/chi-custom-elements/carousel.spec.js (hunk 2 above)
Comment (pedrotaglialatela-lumen) [reply cypress/integration/chi-custom-elements/carousel.spec.js]: Done!

**Review by jllr — APPROVED** [review]

**Pull request merged**